hbase-server/src/main/java/org/apache/hadoop/hbase/io/hfile/LruBlockCache.java

```diff
@@ -299,15 +307,15 @@ public LruBlockCache(long maxSize, long blockSize, Configuration conf) {
    * @param memoryFactor        percentage of total size for in-memory blocks
    */
   public LruBlockCache(long maxSize, long blockSize, boolean evictionThread,
-      int mapInitialSize, float mapLoadFactor, int mapConcurrencyLevel,
-      float minFactor, float acceptableFactor, float singleFactor,
-      float multiFactor, float memoryFactor, float hardLimitFactor,
-      boolean forceInMemory, long maxBlockSize) {
+                       int mapInitialSize, float mapLoadFactor, int mapConcurrencyLevel,
+                       float minFactor, float acceptableFactor, float singleFactor,
+                       float multiFactor, float memoryFactor, float hardLimitFactor,
+                       boolean forceInMemory, long maxBlockSize, int cacheDataBlockPercent) {
```

Comment:
Best to leave these sort of refactorings out of your patch. They bulk it up and distract from what is important about your change. FYI for next time.

Reply:
Thanks! Will correct soon)